tensorflow/lite/kernels/neg_test.cc

```diff
@@ -75,5 +103,61 @@ TEST(NegOpModel, NegInt64) {
   EXPECT_THAT(m.GetOutput<int64_t>(), ElementsAreArray({2, 1, 0, -1, -2, -3}));
 }
 
+class NegOpQuantizedModel : public NegOpModel {
+ public:
+  NegOpQuantizedModel(TensorData input, TensorData output)
+      : NegOpModel(SymmetricInt16Scaling(std::move(input)),
```

Comment:
I think it would be better to just make these constant references

Reply:
Addressed the requested change in [#ee05fdd](https://github.com/MCW-Dev/tensorflow/pull/49/commits/ee05fdd3332fb12a710b5e3d1cdab0b3b3e390ca)